packages/core/src/components/HierarchyPicker/Search/fuseLoader.ts

```diff
@@ -0,0 +1,10 @@
+import Fuse from 'fuse.js';
+
+export default async function fuseLoader(
```

Comment:
We should get rid of this stuff. We can just import fuse directly instead of `import()`ing it.

Reply:
is there any downside to this? there's a lot of code that's based on this assumption sprinkled throughout components and tests.